### DISCUSSION TITLE
Downgrade from 0.4.8

### DISCUSSION THREAD

**Discussion by AdrianChandra** [IceWhaleTech/CasaOS#1755] [Q&A]
Is there anyway to downgrade from 0.4.8 to 0.4.5 (my last known no issue version)?
Cannot edit or install custom app on 0.4.8 (this "chore" release?). 

solution: adding linuxserver.io in app store - more apps solved the issue

**Comment by Raggeman** [comment]
It is the same problem qith 0.4.9 you cannot submit a custom docker app. the submit button does not work

BR/Ulf
